CHANGELOG.next.asciidoc

```diff
@@ -268,6 +268,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Release kvm module as beta. {pull}10279[10279]
 - Release http.server metricset as GA. {pull}10240[10240]
 - Release Nats module as GA. {pull}10281[10281]
+- Release munin module as GA. {pull}10311[10311]
```

Comment:
I am still not 100% sure that current implementation is a final one, but I feel more confident that at least the settings and the field mappings are better than before.

Reply:
We can still improve it moving forward but keep compatibilty.